docs/sources/datasources/tempo/_index.md

```diff
@@ -26,6 +26,10 @@ For instructions on how to add a data source to Grafana, refer to the [administr
 Only users with the organization administrator role can add data sources.
 Administrators can also [configure the data source via YAML](#provision-the-data-source) with Grafana's provisioning system.
 
+This video explains how to add data sources, including Loki, Tempo, and Mimir, to Grafana and Grafana Cloud. Tempo data source set up starts at 4:58 in the video.
+
+{{< youtube id="cqHO0oYW6Ic" >}}
```

Comment:
Same question as in https://github.com/grafana/website/pull/17547/files#r1466714511

Reply:
I'll add the reference here and try the video time.

Reply:
I tried this in the Grafana docs in the other PR by adding the line below and it had no effect. I'm not going to try and add the time stamp here. This is the line I used: 
`{{< youtube id="cqHO0oYW6Ic" start="298" >}}`